gptables/test/test_utils/excel_comparison_test.py

```diff
@@ -0,0 +1,55 @@
+###############################################################################
```

Comment:
can we add a neater join of this onto the backlog - figure out how to make sure we're up to date

Reply:
Ticket in the backlog to improve how we access these files